subcommands/targets/offline-update.go

```diff
@@ -142,7 +142,10 @@ func getTargetInfo(factory string, targetName string) (*ouTargetInfo, error) {
 }
 
 func downloadTufRepo(factory string, tag string, prod bool, dstDir string) error {
-	ver := 1
+	// v1 - auto-generated by tuf_keyserver (default, on Factory creation);
+	// v2 - auto-generated by ota-lite to take keys online (default, on Factory creation);
+	// v3 - customer takes keys offline (prerequisite for offline update);
+	ver := 4
```

Comment:
what error message do we get if ver 4 doesn't exist?

Reply:
None, since this is a normal case, the first three versions are preprovisioned on a device.